plots/dynamics/free_fall_acceleration.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python3
+from sympy.physics.units import gravitational_constant
+from sympy import solve, pretty, symbols
+from sympy.plotting import plot
+from sympy import pretty
+from sympy.plotting import plot
+from sympy.plotting.plot import MatplotlibBackend
+from symplyphysics.laws.dynamics import free_fall_acceleration_from_height as acceleration
+from symplyphysics import (
+    units, convert_to, SI, errors
```

Comment:
SI and errors are unused. Use vscode to highlight unused imports and remove them.